src/components/LinterMessage/styles.module.scss

```diff
@@ -1,3 +1,43 @@
+$top-arrow-width: 10px;
+$top-arrow-height: 20px;
+
+.inline:first-child {
+  &:after,
+  &:before {
+    border-color: transparent;
+    border-style: solid;
+    content: '';
+    display: block;
+    height: 0;
+    left: 10px;
+    position: absolute;
```

Comment:
This relies on `position: relative` within `.alert` which is upstream in react-bootstrap as far as I can tell. I suggest making this explicit in our own styles.